src/xcode/ENA/ENA/Source/AppDelegate & Globals/__tests__/ENAUITestsQuickActions.swift

```diff
@@ -159,14 +160,17 @@ class ENAUITestsQuickActions: XCTestCase {
 
 		// Back on home screen?
 		XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Home.submitCardButton].waitForExistence(timeout: .medium))
-		try checkAppMenu(expectNewDiaryItem: true) // available again?
+		try checkAppMenu(expectNewDiaryItem: true, expectEventCheckin: true) // available again?
 	}
 
 
 	/// Checks the state of the quick action menu according to given parameter.
-	/// - Parameter expectNewDiaryItem: The desired state wether the menu item is existing or not.
+	///
+	///  Foo ba fs f we
```

Comment:
what's this?

Reply:
Oh… a test that slipped through the filter :D